tests/serialization_tests.rs

```diff
@@ -63,6 +63,90 @@ fn serialize_felts(felts: &[gnark_backend_wrapper::Fr]) -> Vec<u8> {
     buff
 }
 
+fn ping_pong(
+    encoded_element: String,
+    go_function: unsafe extern "C" fn(gnark_backend_wrapper::GoString) -> *const ffi::c_char,
+) -> &'static [u8] {
+    // Prepare ping for Go.
+    let pre_ping = ffi::CString::new(encoded_element).unwrap();
+    let ping = gnark_backend_wrapper::GoString::try_from(&pre_ping).unwrap();
+
+    // Send and receive pong from Go.
+    let pong: *const ffi::c_char = unsafe { go_function(ping) };
+
+    // Prepare pong for Rust.
+    let go_pre_serialized_element = unsafe { ffi::CStr::from_ptr(pong) };
+
+    go_pre_serialized_element.to_str().unwrap().as_bytes()
+}
+
+fn random_add_term() -> AddTerm {
+    // Sample random coefficient.
+    let coefficient: gnark_backend_wrapper::Fr = rand::random();
+    // Sample a random sum.
+    let sum = acvm::Witness::new(rand::random());
+    AddTerm { coefficient, sum }
```

Comment:
Maybe this one would require a `cargo fmt`.
```suggestion
    AddTerm { coefficient: rand::random() , sum: acvm::Witness::new(rand::random()) }
```